test/Faker/Provider/en_GB/CompanyTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Faker\Provider\en_GB;
+
+use Faker\Test\TestCase;
+
+/**
+ * @group legacy
+ */
+final class CompanyTest extends TestCase
+{
+    public function testVat()
+    {
+        $number = $this->faker->vat();
+        self::assertEquals(1, preg_match('/^GB[\d]{3} [\d]{4} [\d]{2}$/', $number));
+    }
+
+    public function testVatDefaultType()
+    {
+        $this->testVat();
```

Comment:
Useless method

Reply:
sorry, not sure why I left it like that! Have adjusted to reflect the original intention